ts-shared/viz-expr/eval-on-backend.ts

```diff
@@ -10,7 +10,7 @@ export const EvalAppRequestParams = Schema.Struct({
   $type: Schema.tag('EvalAppRequestParams'),
   // We can think about batching requests in the future
   appExpr: IRId,
-  arguments: Schema.Array(BoolValue).annotations({
+  args: Schema.Array(BoolValue).annotations({
```

Comment:
Refactor: Rename `EvalAppRequestParams`' `arguments` fieldname to `args` for JS reasons